### ISSUE TITLE
Does the edge loss help?

### ISSUE THREAD

**Issue by IssamLaradji** [DengPingFan/Inf-Net#13]
I tried with and without the edge loss and I didn't see a difference in the results. Did you find that the edge loss helps?

**Comment by GewelsJI** [comment]
First, I don't check your code, have you disable your edge-aware loss successfully? Second, edge loss can help depict more fine-grained boundaries in my empirical, and thus, it will improve performance slightly. You better try to compare different qualitative results under different version (with or without edge loss) or retrain it again after you check your code. Thank you for your attention.

**Comment by IssamLaradji** [comment]
You welcome.

Yes same results without edge-aware loss. I also used plain FCN8 and UNet and I get the same results in terms of DSC score. I used the the evaluation metric you provided.

**Comment by DengPingFan** [comment]
Dr. IssamLaradji 
For the EA loss, we have conducted the ablation study in TABLE IV, as we provide in the manuscript. It has a slight improvement for the Sm score (from 0.651 to 0.673). This due to we only have 50 training images. If you provide more data for training our network, it will provide more performance imp. In addition, the EA loss will accelerate the convergence speed of the network. That is why our model only needs 15 minutes to converge over 100 epochs. We are happy if you could send us your training code. 

Best, Deng-Ping

**Comment by DengPingFan** [comment]
Dr. IssamLaradji
For the EA loss, we have conducted the ablation study in TABLE IV, as we provide in the manuscript. It has a slight improvement for the Sm score (from 0.651 to 0.673). This due to we only have 50 training images. If you provide more data for training our network, it will provide more performance imp. In addition, the EA loss will accelerate the convergence speed of the network. That is why our model only needs 15 minutes to converge over 100 epochs. We are happy if you could send us your training code.

Best, Deng-Ping
